### ISSUE TITLE
Reach The Standing Stones not updating.

### ISSUE THREAD

**Issue by MoonlitRiverlight** [chaosvolt/cdda-arcana-mod#243]
Reached Stonehenge with the NPC, no change. Might've expired (it's been months since I've picked it) as the NPC came up with a generic quest upon being prompted.
Any solutions I could try or is this a dead playthrough?

**Comment by chaosvolt** [comment]
That's odd. He doesn't follow up with another mission to visit a ruined shrine? It doesn't have an expiration date set, so. In any case, in the debug menu you can edit players/NPCs, and you should be able to add missions to the NPC, the next one in the mission chain is "Reach Ruined Shrine"

**Comment by MoonlitRiverlight** [comment]
Nope, doesn't even recognize we're there. 
I managed to bypass by removing that quest from both of us then assigning it to the NPC again and taking it over, at which point it did recognize we were standing on top of it.

**Comment by chaosvolt** [comment]
Did you make sure that both you and the NPC were standing inside the specific map tile that the standing stones are in?

**Comment by MoonlitRiverlight** [comment]
Yes. We were standing right by the brazier. On reassigning the quest it got completed without having to move.

**Comment by chaosvolt** [comment]
Never was able to reproduce this, and repo is being archived due to it now being an in-repo mod in BN.
